### DISCUSSION TITLE
Render Sprite Sheet Comparison out

### DISCUSSION THREAD

**Discussion by arfx** [ggarra13/mrv2#300] [Ideas]
I use mrv2 quick often for comparison of multiple PNG Image Sequences, and sometimes I make some Annotations into them. 
I would like to render all of these as a one single new Video out.

![mrv2_2024-09-10_15-36-16](https://github.com/user-attachments/assets/c07ba163-e352-4567-97a5-3bf31757ad22)

**Comment by ggarra13** [comment]
Your wish is my command, sir! 👍 

Windows .exe:
https://mega.nz/file/yb5mVQwL#Gf_xdNT-NzSqGQj6ZNDtmFK2RfvZtJf4PABHzAkDdv0

Windows .zip:
https://mega.nz/file/uPpzSS7Z#RwYG-jJtduqpG7RZcsblSaSRwLIMopl-XHo0fs0lFbg


https://github.com/user-attachments/assets/2000299c-7b5f-480e-bb87-b66b48185ec4

**Comment by arfx** [comment]
You´re awesome dude and always so fast, thank you. 🙏

I just noticed, that when I have multiple viewed and don´t checked "Save Annotations" it doesn´t work. 
Maybe you can add a separated toggle in the export window for that, to export the whole view. 

https://github.com/user-attachments/assets/002d9672-bf4d-4031-9254-44f68f3eb4b9

**Reply by ggarra13** [reply]
> I just noticed, that when I have multiple viewed and don´t checked "Save Annotations" it doesn´t work.
> Maybe you can add a separated toggle in the export window for that, to export the whole view.

I am keeping it as is, but added some more documentation to the tooltip of Save Annotations.
